<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="toolTip">
   <string extracomment="0"/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>471</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>540</width>
      <height>364</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>255</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string notr="true">0</string>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::LogicalMoveStyle</enum>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="pushButton_eq">
             <property name="text">
              <string>=</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="pushButton_minus">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="pushButton_mult">
             <property name="text">
              <string>*</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="pushButton_dot">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="pushButton_plus">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButton_1">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="pushButton_div">
             <property name="text">
              <string>/</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButton_9">
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButton_0">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_cos">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>cos</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_sin">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>sin</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButton_tan">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>tan</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="pushButton_sqrt">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>sqrt</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_acos">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>acos</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_asin">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>asin</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_atan">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>atan</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="pushButton_ln">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>ln</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButton_log">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>log</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_brck_cl">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="pushButton_pow">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>^</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="pushButton_brck_op">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>(</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButton_mod">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>mod</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="pushButton_var_x">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="pushButton_clear">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="pushButton_del">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>AC</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Xmin</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_Xmin">
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <double>-10000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>10000000.000000000000000</double>
             </property>
             <property name="value">
              <double>-10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Xmax</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_Xmax">
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <double>-10000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>10000000.000000000000000</double>
             </property>
             <property name="value">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_graph">
         <property name="text">
          <string>Plot graph</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton_debit">
         <property name="text">
          <string>Debit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_credit">
         <property name="text">
          <string>Credit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_cos</tabstop>
  <tabstop>pushButton_tan</tabstop>
  <tabstop>pushButton_sqrt</tabstop>
  <tabstop>pushButton_sin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
